devicemodel/core/main.c

```diff
@@ -1020,6 +1028,18 @@ main(int argc, char *argv[])
 			usage(1);
 		}
 	}
+
+	if (get_iasl_compiler() != 0) {
+		pr_err("Cannot find Intel ACPI ASL compiler tool \"iasl\".\n");
+		exit(1);
+	}
+
+	if (check_iasl_version() != 0) {
+		pr_err("Please install iasl tool with version >= %s from https://www.acpica.org/downloads \n",
+			IASL_MIN_VER);
```

Comment:
How about being more verbose here and also tell the user that he should install the min version and provide the path to it if it's not on the PATH?

We could also run into the situation whereby `iasl` is on the system (was installed via the `acpica-tools` package) and another version compiled by the user is also installed but in a different path (e.g. `/usr/sbin` vs. `/usr/bin`). In that case, the error message can be confusing because it will complain that the version is too old since it's only looking at one of them.

Reply:
I have updated the message.